ppa/archive/forms.py

```diff
@@ -25,9 +25,31 @@ def valid_value(self, value):
 
 class SearchForm(forms.Form):
     '''Simple search form for digitized works.'''
+
+    SORT_CHOICES = [
+        ('relevance', 'relevance'),
+        ('pub_date_asc', 'chronology (earliest to latest)'),
+        ('pub_date_desc', 'chronology (latest to earliest)'),
```

Comment:
these have different human-friendly names (e.g. "Year Oldest-Newest") in the design spec. I'm already editing this stuff for a frontend PR so I can just change it there, though.

Reply:
@thatbudakguy No reason for me not to nab it now. It'll save you some copy-paste later. Thanks for the spot.

Reply:
speaking of copy-paste, I already had this snippet in there:
```
    SORT_TYPES = {
        'relevance': 'Relevance',
        'year_oldest': 'Year Oldest-Newest',
        'year': 'Year Newest-Oldest',
        'title': 'Title A-Z',
        'title_z': 'Title Z-A',
        'author': 'Author A-Z',
        'author_z': 'Author Z-A'
    }
```
maybe saves you 30sec because the keys are wrong, but whatever

Reply:
funny that we both filled in opposite sides of a virtually identical dict without knowing...